content/en/api/overview.md

```diff
@@ -5,6 +5,21 @@ position: 40
 category: API
 ---
 
+## URLs
+
+To create a Checkout Session you need to call the API with the following endpoints:
```

Comment:
This API is generic, not just for Checkout.

Something like "The API endpoints are available in the following domains:" ?

Reply:
Yes, where is it mentioning that it's not generic? What is the difference from `..call the API with the following endpoints` to `The API endpoints are available in the following domains:`? The domains are explicit under the `You have 2 options:` part.

Reply:
"To create a Checkout Session" implies that it's only for Checkout.

The difference of that part is that what you're listing below are not endpoints, just the domains that will prefix the endpoints.

Endpoints will be listed in each specific service (e.g. `/checkout`).

Reply:
I see what u mean, this is the `api overview`, not the `checkout overview`.